index.html

```diff
@@ -49,6 +49,7 @@
     <div id='nav'>
       <div class='container center'>
         <a href="#contribute">Contribute</a>
+        <a href="#ci">CI</a>
```

Comment:
To tell you the truth -- I was thinking exactly that, but then I realized CI is just two characters ;-) But seriously enough I *really* would like CI links to be easily discoverable same way they are right at the top in our GitHub README. Let me know if you feel super strongly about *not* allowing CI into the top bar.